djangocms_versioning_filer/helpers.py

```diff
@@ -49,3 +52,25 @@ def is_moderation_enabled():
         )
     except LookupError:
         return False
+
+
+def check_file_label_exists_in_folder(label, folder, exclude_file_pks=None):
+    if not isinstance(exclude_file_pks, collections.Iterable):
```

Comment:
Can we use this instead?
```suggestion
    if exclude_file_pks is None:
```